pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -84,7 +84,9 @@ func NewCmdRollingUpdate(f cmdutil.Factory, out io.Writer) *cobra.Command {
 		Short:   i18n.T("Perform a rolling update of the given ReplicationController"),
 		Long:    rollingUpdateLong,
 		Example: rollingUpdateExample,
+		Deprecated = fmt.Sprintf("use kube instead", originalName)
```

Comment:
Definitely not my role to review this, so sorry if this is intended but you are missing a `%s` or `%v` here.

Reply:
Good catch, thanks!

Reply:
I actually went with `printDeprecationWarning` instead so this line is unnecessary.